Dockerfile

```diff
@@ -31,6 +31,8 @@ RUN ./manage.py collectstatic --noinput
 RUN apt-get clean && apt-get purge
 USER mitodl
 
+RUN mkdir /var/media && chown -R mitodl:mitodl /var/media
```

Comment:
Just running `docker-compose build` on my machine locally it seems like the `mitodl` user doesn't have permission to create this directory.